protocol/x/clob/keeper/clob_pair.go

```diff
@@ -165,19 +168,6 @@ func (k Keeper) createOrderbook(ctx sdk.Context, clobPair types.ClobPair) {
 // CreateClobPair creates a new `ClobPair` in the store and creates the corresponding orderbook in the memclob.
```

Comment:
nit: Update the comment / naming to indicate this doesn't create a new ClobPair in the store.